From 864830dbd2293d986349d633940f9d03e3407844 Mon Sep 17 00:00:00 2001
From: Lubomir Rintel <lkundrak@v3.sk>
Date: Fri, 30 Sep 2011 09:38:45 +0200
Subject: [PATCH 1/3] Remove __unused

find SDKs/MacOSX10.6.sdk/usr/include/{i386,architecture,mach{,-o,_debug,ine},sys/_types.h} -name '*.h' -type f |
	xargs sed 's/__unused/__attribute__((__unused__))/g' -i
---
 SDKs/MacOSX10.6.sdk/usr/include/mach/mig_errors.h |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/SDKs/MacOSX10.6.sdk/usr/include/mach/mig_errors.h b/SDKs/MacOSX10.6.sdk/usr/include/mach/mig_errors.h
index f1234bc..df73840 100644
--- a/SDKs/MacOSX10.6.sdk/usr/include/mach/mig_errors.h
+++ b/SDKs/MacOSX10.6.sdk/usr/include/mach/mig_errors.h
@@ -110,7 +110,7 @@ __BEGIN_DECLS
 #define __NDR_convert__mig_reply_error_t__defined
 
 static __inline__ void
-__NDR_convert__mig_reply_error_t(__unused mig_reply_error_t *x)
+__NDR_convert__mig_reply_error_t(__attribute__((__unused__)) mig_reply_error_t *x)
 {
 #if defined(__NDR_convert__int_rep__kern_return_t__defined)
 	if (x->NDR.int_rep != NDR_record.int_rep)
-- 
1.7.1

